cabal-testsuite/main/cabal-tests.hs

```diff
@@ -83,7 +83,7 @@ main = do
     hSetBuffering stderr LineBuffering
 
     -- Parse arguments
-    args <- execParser (info mainArgParser mempty)
+    args <- execParser $ info (mainArgParser <**> helper) mempty
```

Comment:
What does the `helper` do? Perhaps worth a tiny comment?

Reply:
`helper` adds option `--help` and thus the possibility to display detailed help.  (Otherwise you never get to see the `help` text supplied in the option definitions.)  I can write a comment...